Tables/External Tables.event

```diff
@@ -8,6 +8,10 @@ defWeaponDebuffs(4,0,0,4,0,4,4) //0x5: Pebble debuff.
 
 ALIGN 4
 
+ArmorFlyingMagicEffectiveness:
+BYTE $09 $0A $0B $0C $35 $36
+FlyingMagicEffectiveness:
+BYTE $48 $49 $1F $20 $21 $22 $23 $24
```

Comment:
Since we use these multiple times, can you make a definition
```
#define FlyingClasses "$48 $49 $1F $20 $21 $22 $23 $24"
```
And use that here and below? Same for Armor.